src/views/Timeline.vue

```diff
@@ -464,7 +476,7 @@ export default {
       if (inf.length > 0) {
         topLabel = inf[0].label
       }
-      let icon = 'mdi-crosshairs-question'
+      let icon = 'mdi-karate'
```

Comment:
@vickywane does this still need to be here ?

Reply:
Should we pull icon choice into its own `ambianic-event-view` component outside of the timeline view page?

Reply:
I have removed the first icon value. The icon from Iconfiy would be used for the fall icon.

Reply:
OK. Good work moving that out into an `eventIcon` component.